modules/oc-adm-by-example-content.adoc

```diff
@@ -346,6 +350,96 @@ Display and filter node logs
 
 
 
+== oc adm ocp-certificates monitor-certificates
+Watch platform certificates.
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Watch platform certificates.
+  oc adm ocp-certificates monitor-certificates
+----
+
+
+
+== oc adm ocp-certificates regenerate-leaf
+Regenerate client and serving certificates of an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known leaf certificates on the cluster.
+  oc adm certificates regenerate-leaf -A --all
+  
+  # Regenerate a leaf certificate contained in a particular secret.
+  oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
+----
+
+
+
+== oc adm ocp-certificates regenerate-machine-config-server-serving-cert
+Regenerate the machine config operator certificates in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate the MCO certs without modifying user-data secrets
+  oc adm certificates regenerate-machine-config-server-serving-cert --update-ignition=false
+  
+  # Update the user-data secrets to use new MCS certs
+  oc adm certificates update-ignition-ca-bundle-for-machine-config-server
+----
+
+
+
+== oc adm ocp-certificates regenerate-top-level
+Regenerate the top level certificates in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known signing certificates on the cluster.
+  oc adm certificates regenerate-signers -A --all
+  
+  # Regenerate the signing certificate contained in a particular secret.
+  oc adm certificates regenerate-signers -n openshift-kube-apiserver-operator loadbalancer-serving-signer
+----
+
+
+
+== oc adm ocp-certificates remove-old-trust
+Remove old CAs from ConfigMaps representing platform trust bundles in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Remove all known trust bundles in the cluster
+  oc adm ocp-certificates remove-old-trust configmaps -A --all
+  
+  # Remove a trust bundled contained in a particular config map
+  oc adm ocp-certificates remove-old-trust -n openshift-config-managed configmaps/kube-apiserver-aggregator-client-ca
```

Comment:
`oc adm ocp-certificates remove-old-trust -n openshift-config-managed configmaps/kube-apiserver-aggregator-client-ca
error: required flag(s) "created-before" not set`

Reply:
Removing this example temporarily